src/docs/content/docs/_index.md

```diff
@@ -4,5 +4,25 @@ show_menu = true
 
 ## Welcome to the Cassandra Reaper documentation!
 
+### Overview
 
+Reaper is an open source tool that aims to schedule and orchestrate repairs of Apache Cassandra clusters. It improves on a repair process based on the classic nodetool repair by allowing
```

Comment:
> It improves on a repair process based on the classic nodetool repair by allowing

newline first, then

It improves the existing nodetool repair process by
* Splitting repair jobs into smaller tunable segments.
* Handling back-pressure through monitoring running repairs and pending compactions.
* Adding ability to pause or cancel repairs and track progress precisely.

Reply:
Agreed.